core/frontend/src/Viewport.ts

```diff
@@ -158,18 +158,6 @@ export interface ChangeViewedModel2dOptions {
  */
 export enum ViewUndoEvent { Undo = 0, Redo = 1 }
 
-/** Options controlling display of [OpenStreetMap Buildings](https://cesium.com/platform/cesium-ion/content/cesium-osm-buildings/).
- * @see [[Viewport.setOSMBuildingDisplay=]].
-   * @public

```

Comment:
Is this a breaking change for public API?

Reply:
It moved to DisplayStyleState.ts - is unused in Viewport.ts.

Reply:
Please resolve comments if addressed to your satisfaction - PRs cannot be completed until all comments are resolved.